app/components/candidate_interface/equality_and_diversity_review_component.rb

```diff
@@ -59,5 +59,31 @@ def ethnicity_row
         },
       }
     end
+
+    def free_school_meals_row
+      return if not_answered_free_school_meals?
+
+      free_school_meals = if @application_form.equality_and_diversity['free_school_meals'] == 'no'
+                            t('equality_and_diversity.free_school_meals.no.review_value')
+                          elsif @application_form.equality_and_diversity['free_school_meals'] == 'yes'
+                            t('equality_and_diversity.free_school_meals.yes.review_value')
+                          elsif @application_form.equality_and_diversity['free_school_meals'] == 'I do not know'
+                            t('equality_and_diversity.free_school_meals.unknown.review_value')
+                          else
+                            @application_form.equality_and_diversity['free_school_meals']
+                          end
+      {
+        key: 'Free school meals',
+        value: free_school_meals,
+        action: {
+          href: candidate_interface_edit_equality_and_diversity_free_school_meals_path,
+          visually_hidden_text: 'free school meals',
```

Comment:
Think it'd be better to use slightly more verbose visually-hidden text here.
```suggestion
          visually_hidden_text: 'whether you ever got free school meals',
```